actionpack/lib/action_controller/caching.rb

```diff
@@ -46,6 +46,20 @@ def cache_configured?
         end
     end
 
+    # Cache or yield the block. The cache is suppose to never expire.
```

Comment:
Maybe should I move this to https://github.com/rails/rails/blob/master/actionpack/lib/action_controller/metal/conditional_get.rb ?